normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
+        schema = json.loads(
+            open('normandy/recipes/static/actions/console-log/package.json').read()
+        )
+
+        ActionFactory(
+            name='console-log',
+            arguments_schema=schema['normandy']['argumentsSchema']
+        )
+
+        # If the action specified cannot be found, raise validation
+        # error indicating the arguments schema could not be loaded
+        serializer = RecipeSerializer(data={
+            'action': 'action-that-doesnt-exist', 'arguments': {}
+        })
+
+        serializer.is_valid()
+        assert serializer.errors['arguments'] == ['Could not find arguments schema.']
+        assert pytest.raises(serializers.ValidationError)
```

Comment:
I don't think this does what you want it to. Usually this is used as

```python
with pytest.raises(SomeExceptionType):
    something_that_raises_that_type()
```

Just calling `pytest.raises(...)` alone doesn't actually assert anything.